pkgs/tools/misc/hexbinhex/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hexbinhex";
+  version = "unstable-2023-03-31";
+
+  src = fetchFromGitHub {
+    owner = "dj-on-github";
+    repo = "${pname}";
+    rev = "41802864f4d281a852a5dc83d377a83013264485";
+    hash = "sha256-ODHaVG0mEQ7yY0iNJOudU08ElxqEiyr4oHAHRrlpd28=";
+  };
+
+  preBuild = lib.optional stdenv.hostPlatform.isAarch64 ''
+    sed -i s/gcc/${stdenv.cc.targetPrefix}gcc/g Makefile
+    sed -i s/-m64//g Makefile
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp ./bin2hex $out/bin
+    cp ./bin201 $out/bin
+    cp ./012bin $out/bin
+    cp ./hex2bin $out/bin
+    cp ./bin2nistoddball $out/bin
+    cp ./nistoddball2bin $out/bin
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/dj-on-github/hexbinhex";
+    description = ''Six utility programs to convert between hex, binary, ascii-binary
+      and the oddball NIST format for 90B testing.'';
```

It is more common to see this style of line wrapping in Nixpkgs. This is however just a nit. Don't feel obligated to change it.
```suggestion
    description = ''
      Six utility programs to convert between hex, binary, ascii-binary
      and the oddball NIST format for 90B testing.
    '';
```

---

I have changed it how you suggested. This does look better.